assets/js/localization.js

```diff
@@ -411,6 +439,10 @@ function getLangByCode(code) {
     }
 }
 
+function isRtlLanguage(lang) {
```

Comment:
Wait, if there's already a `direction` function, then you should simply use that. There's no use to making two functions that do essentially the same thing.